dev/src/codewind/connection/UserProjectCreator.ts

```diff
@@ -120,7 +120,35 @@ namespace UserProjectCreator {
             projectTypeInfo = userProjectType;
         }
 
-        await requestBind(connection, projectName, pathToBind, projectTypeInfo.language, projectTypeInfo.projectType);
+        return bind(connection, projectName, pathToBind, projectTypeInfo);
+    }
+
+    async function bind(connection: Connection, projectName: string,
+                        pathToBind: string, projectTypeInfo: IProjectTypeInfo):
+                        Promise<INewProjectInfo | undefined> {
+        if (connection.remote) {
+            return bindRemote(connection, projectName, pathToBind, projectTypeInfo);
+        } else {
+            return bindLocal(connection, projectName, pathToBind, projectTypeInfo);
```

Comment:
dead code because `remote` is always true?

Reply:
Yes, I'm going to need to remove the dead paths here and (more so) in pfe. (I need to remove the initialise container there.) I left them in for now as it makes this PR much easier to merge.